Client/Wallet/EthereumProviderHelper.swift

```diff
@@ -80,6 +80,12 @@ class EthereumProviderHelper: TabContentScript {
       return
     }
     
+    if message.webView?.url?.isLocal == false,
+        message.webView?.hasOnlySecureContent == false { // prevent communication in mixed-content scenarios
+      replyHandler(nil, "Failed Security Test")
```

Comment:
No need for the `replyHandler`. We're going with leaving the `promise` unresolved.

Reply:
Updated in 05052e1.